src/NUnitFramework/Directory.Build.props

```diff
@@ -32,4 +32,8 @@
     <EmbedAllSources>true</EmbedAllSources>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="DotNetAnalyzers.DocumentationAnalyzers" Version="1.0.0-beta.59"/>
```

Comment:
Sorry for not mentioning. This should have `PrivateAssets="all"` in case we ever use `dotnet pack` to create our NuGet packages. Otherwise the NUnit package will have a .nuspec dependency on `DotNetAnalyzers.DocumentationAnalyzers`, causing any project that references NUnit to transitively reference DocumentationAnalyzers.

Reply:
Tracked at https://github.com/nunit/nunit/pull/3551.